R/get_generic.R

```diff
@@ -0,0 +1,72 @@
+## function name get_generic
+
+#' retrieve all parameters of specified type
+#' @param data parameter dataframe outputted from load_epidata()
+#' @param parameter_name name of the parameter type to retrieve, ensuring the name matches that in data
+#' @return dataframe with all parameter estimates and specified columns
+#' @examples
+#' df <- load_epidata(pathogen = "ebola")
+#' get_generic(data = df$params, parameter_name = "Human delay - serial interval")
+#' @examples
+#' df <- load_epidata(pathogen = "marburg")
+#' get_generic(data = df$params, parameter_name = "Attack rate")
+get_generic <- function(data, parameter_name) {
```

Comment:
Suggest renaming this to a more informative function name. What about get_parameter?